SplitWise/src/com/workattech/sw/models/Money.java

```diff
@@ -0,0 +1,38 @@
+package com.workattech.sw.models;
+
+public class Money {
+       String personA;
+       String personB;
+       double amount;
+       public Money(String moneyGivenBy,String moneyTakenBy,double amount)
+       {
+    	   this.personA=moneyGivenBy;

```

Comment:
Money should not know about who paid to whom. A better name would have been expense